platform/node/src/node_thread_pool.cpp

```diff
@@ -0,0 +1,38 @@
+#include "node_thread_pool.hpp"
+#include "util/async_queue.hpp"
+
+#include <mbgl/actor/mailbox.hpp>
+
+namespace node_mbgl {
+
+NodeThreadPool::NodeThreadPool()
+    : queue(new util::AsyncQueue<std::weak_ptr<mbgl::Mailbox>>(uv_default_loop(), [this](std::weak_ptr<mbgl::Mailbox> mailbox) {
```

Comment:
Ack, needed to fix this in [`mapbox-gl-native/platform/node/src/node_log.cpp`](https://github.com/mapbox/mapbox-gl-native/blob/1898df606dd6ca3c8badcaa40cb17c2bce307055/platform/node/src/node_log.cpp#L22) too.